homeassistant/components/flume/sensor.py

```diff
@@ -152,7 +156,11 @@ def update(self):
             self._available = False
             return
         _LOGGER.debug("Successful update of flume sensor: %s", self._name)
-        self._state = self._flume_device.value
+        self._state = self._flume_device.values["current_interval"]
+        self._attributes = {
+            k: round(self._flume_device.values[k], 1)
+            for k in (self._flume_device.values.keys() - ["current_interval"])
```

Comment:
We use `.items()` to iterate the dict.

Reply:
Removed this loop as it's no longer needed when setting attributes.